cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllMenu/index.tsx

```diff
@@ -52,7 +52,12 @@ const styles = (theme): StyleRules => {
   };
 };
 
-function FllMenu({ classes }) {
+interface IFllMenuProps extends WithStyles<typeof styles> {
```

Comment:
is this interface being used?

Reply:
Oops, forgot to actually use it here. Have fixed to actually use the interface.